login.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+     include_once('header.php'); // Include Header once
+     
+if (isset($_POST['login_user'])) {
+    $errors = array();
+    $first_name = '';
+    $email_address = mysqli_real_escape_string($db_connect, $_POST['email_address']);
+    $user_password = mysqli_real_escape_string($db_connect, $_POST['user_password']);
+    
+    if (empty($email_address)) {
+          array_push($errors, 'Email address is required');
+    }
+    if (empty($user_password)) {
+        array_push($errors, 'Password is required');
+    }
+    if (count($errors) == 0) {
+        $user_password = md5($user_password);
+    
+        $query = "SELECT * FROM users WHERE email = '$email_address' AND user_password = '$user_password'";
+    
+    
+        $results = mysqli_query($db_connect, $query);
+         
+        $matches = mysqli_num_rows($results);
+        if (0 !== $matches) {
+            while ($row = $results->fetch_row()) {
+                 //var_dump($row);
+                 $first_name = $row[1];
+            }
+        }
+    
+        if (mysqli_num_rows($results)) {
+            $_SESSION['email'] = $email_address;
+            $_SESSION['success'] = "Logged in successfully $first_name";
+            echo "$first_name logged in .";
+            $_SESSION['login'] = 1;
```

Comment:
There is no point in having this `$_SESSION['login']` variable.

What I can suggest it to SELECT the id of the user from the database and save it instead.
You will later need the ID of the user for the dashboard. Aaaand, in order to check if the user is logged, all if you have to do is check if the ID is set in the `$_SESSION` variable